### PR TITLE
test against couch 2

### PR DESCRIPTION
**Pull request by emord**
Realized we've never actually tested this on couch 2. Here's some fun failures:

`reason":"all_or_nothing is not supported"}`
`Unauthorized: {"error":"forbidden","reason":"Temporary views are not supported in CouchDB"}`
`PreconditionFailed: The database could not be created, the file already exists.`

@dannyroberts @snopoke

### CODE REVIEW

**Comment by snopoke** [comment]
What's your plan for this?

**Comment by emord** [comment]
No. I plan on just ripping out the parts that don't work in couch 2, since we don't use that anymore (except swiss, which should get moved at some point)

I plan on working on https://github.com/dimagi/couchdbkit/pull/24 for the hackathon and realized that I had removed some tests there because they don't actually work in couch 2

**Comment by emord** [comment]
Ripped out stuff that's specific to couch 1, and fixed some tests. 

There's a doc folder that looks autogenerated. guessing it powered (now defunct) couchdbkit.org. There aren't any docs about generating the docs. should we kill that folder too?

File: tests/client_test.py [hunk 1]
```diff
@@ -497,61 +497,6 @@ def testDeleteMultipleDocs(self):
 
         del self.Server['couchdbkit_test']
 
-    def testMultipleDocCOnflict(self):
```
Comment (snopoke) [inline tests/client_test.py]: what's wrong with this test? Seems like it's just the last bit that should be removed (with `all_or_nothing`)

Comment (emord) [reply tests/client_test.py]: good point https://github.com/dimagi/couchdbkit/pull/26/commits/8544e945d145f5b6ac7b8cd3452be1982355e120

**Comment by emord** [comment]
I think this is good to go

**Pull request merged**